src/components/HomePage/FindSection.tsx

```diff
@@ -1,18 +1,11 @@
-import * as React from 'react';
+import React, { FC } from 'react';
 import { Input, InputGroup } from 'reactstrap';
-import AppButton from '../shared/Buttons/AppButton';
-import Heading from '../shared/Heading';
-
-const FindSection: React.FC<{}> = () => {
-  const addMovieClick = () => {
-    const addMoviePopup = document.getElementById('add-movie-popup') as HTMLButtonElement;
-    addMoviePopup.classList.remove('invisible');
-    // todo: add logic when I start task 4 and 7
-  };
+import { AppButton } from '../shared/Buttons/AppButton';
```

Comment:
would be nice to add `index.ts` to `shared` folder and export all shared components.

export {AppButton} from './Buttons'
export {Heading} from './Text'
...

And import it from one directory:
import {AppButton, Heading} from '../shared'

Reply:
Thanks. Added in commit 10558d1.